app/assets/stylesheets/common/base/tagging.scss

```diff
@@ -203,6 +203,14 @@ header .discourse-tag {
   }
 }
 
+.hashtag-autocomplete {
```

Comment:
that feels a little bit dirty, cant we flex the whole thing?

Reply:
I didn't want to mess too much with the template and styles at first, but your comment made me realise it doesn't matter since I am making a new template anyway. Will try some flex things out 👍